packages/core/analytics/core-analytics-browser-internal/src/analytics_service.ts

```diff
@@ -38,6 +39,7 @@ export class AnalyticsService {
 
     this.registerBuildInfoAnalyticsContext(core);
     registerPerformanceMetricEventType(this.analyticsClient);
+    registerSubscriptionTrackingEvents(this.analyticsClient);
```

Comment:
This feels wrong. You're registering inside a core service something exposed from a package owned by another team. It feels like a wrong isolation of concerns?

But I'll defer to @afharo

Reply:
@afharo Yep, makes sense to decide on ownership after this PR gets merged!